packages/shopify-codemod/transforms/constant-function-expression-to-statement.js

```diff
@@ -0,0 +1,30 @@
+export default function constantFunctionValueToStatement(file, api, options) {
+  const j = api.jscodeshift;
+  const {expression, statement, statements} = j.template;
+  const printOptions = options.printOptions || {quote: 'single'};
+
+  return j(file.source)
+    .find(j.VariableDeclaration, (p) => j.match(p, {
```

Comment:
well, with filter I'd also have to do `.type === 'VariableDeclaration'`. This is what gets called from find https://github.com/facebook/jscodeshift/blob/master/src/matchNode.js